docs/4.3/kubernetes_ssh.md

```diff
@@ -129,6 +129,12 @@ configured. In this guide we'll look at two common configurations:
   In this case, we'll need to map users' groups that come from Okta to Kubernetes
   groups.
 
+## Kubernetes Groups and Users
+
+Teleport provides support for Kubernetes Group `kubernetes_groups: ["system:masters"]` 
+and Kubernetes Users, using `kubernetes_users: ['IAM#barent', 'IAM#jane']`. If a Kubernetes
```

Comment:
Why the `IAM#` prefix?

Reply:
We originally wrote this so a customer could pass through identity from their internal identity provider - I think that's how their system returned the usernames and we just haven't made it more "generic".

Reply:
Good call, I've removed for now.